src/number_theory/bernoulli.lean

```diff
@@ -0,0 +1,80 @@
+/-
+Copyright (c) 2020 Johan Commelin. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Johan Commelin
+-/
+
+import data.rat
+import data.fintype
+
+/-!
+# Bernoulli numbers
+
+The Bernoulli numbers are a sequence of numbers that frequently show up in number theory.
+
+For example, they show up in the Taylor series of many trigonometric and hyperbolic functions,
+and also as (integral multiples) of special values of the Riemann zeta function.
+(Note: these facts are not yet available in mathlib)
+
+In this file, we provide the definition,
+and the basic fact (`sum_bernoulli`) that
+$$ ∑_{k < n} \binom{n}{k} * B_k = n, $$
+where $B_k$ denotes the the $k$-th Bernoulli number.
+
+-/
+
+/-- The Bernoulli numbers. -/
```

Comment:
Could you write a more readable definition in the docstring?

Reply:
Done